exercises/collatz-conjecture/tests/collatz-conjecture.rs

```diff
@@ -0,0 +1,32 @@
+extern crate collatz_conjecture;
+
+use collatz_conjecture::*;
+
+#[test]
+fn test_1() {
+    assert_eq!(Ok(0), collatz(1));
+}
+
+#[test]
+#[ignore]
+fn test_16() {
+    assert_eq!(Ok(4), collatz(16));
+}
+
+#[test]
+#[ignore]
+fn test_12() {
+    assert_eq!(Ok(9), collatz(12));
+}
+
+#[test]
+#[ignore]
+fn test_1000000() {
+    assert_eq!(Ok(152), collatz(1000000));
+}
+
+#[test]
+#[ignore]
+fn test_0() {
+    assert_eq!(Err("Only positive numbers are allowed"), collatz(0));
```

Comment:
Depending on the answer to whether the `return Err(...)` should be provided, I would think we would only want to test `is_err` on the returned error, to allow the freedom to choose what the `Err` value is.

Reply:
Agreed, but see my comment above.

Reply:
I changed my mind. Even though we are leaving `return Err("Only positive...")` in, I still say this test should only check `is_err`, since I am considering the habits it may instill, and I don't want to instill the habit of checking exact string error messages.